Assets/VRTK/Prefabs/Internal/Scripts/VRTK_RadialMenu.cs

```diff
@@ -8,8 +8,37 @@ namespace VRTK
     using UnityEngine.UI;
     using UnityEngine.EventSystems;
 
+    /// <summary>
+    /// Angle and Deflection of the user's touch on the touchpad
+    /// </summary>
+    /// <param name="angle">The angle of the touch on the radial menu.</param>
```

Comment:
This line is not the valid place for this XML docs comment 😃

It has to go to be in front of the field itself. If you're using Visual Studio you should be able to just place the cursor on the line before the field and type in `///`. It should automatically fill in the XML docs comment template for fields.

Reply:
I copied the style from UIPointerEventArgs, BodyPhysicsEventArgs, etc which have param comments above the struct definition, I can add a comment to the constructor as well.

Reply:
@Fulby You found a docs bug 😄

@thestonefox The struct docs are wrong, same as before for the enums ([enum PR](https://github.com/thestonefox/VRTK/pull/1465)).